Lib/idlelib/autocomplete_w.py

```diff
@@ -182,16 +182,11 @@ def show_window(self, comp_lists, index, complete, mode, userWantsWin):
         self.userwantswindow = userWantsWin
         self.lasttypedstart = self.start
 
-        # Put widgets in place
         self.autocompletewindow = acw = Toplevel(self.widget)
-        # Put it in a position so that it is not seen.
-        acw.wm_geometry("+10000+10000")
-        # Make it float
+        acw.withdraw()
         acw.wm_overrideredirect(1)
         try:
-            # This command is only needed and available on Tk >= 8.4.0 for OSX
-            # Without it, call tips intrude on the typing process by grabbing
-            # the focus.
+            # Prevent grabbing focus on maxOS.
```

Comment:
```suggestion
            # Prevent grabbing focus on macOS.
```

Reply:
I missed this when looking as the pictures below.  Fixed in #104598.